src/test/ui/consts/const_discriminant.rs

```diff
@@ -1,14 +1,9 @@
 // run-pass
-#![feature(const_discriminant)]
+#![feature(const_discriminant, test)]
```

Comment:
```suggestion
#![feature(const_discriminant)]
#![feature(test)]
```

Reply:
No preference from my side, so why not.^^